tyrian/js/src/main/scala/tyrian/runtime/TyrianRuntime.scala

```diff
@@ -139,36 +138,51 @@ final class TyrianRuntime[F[_]: Async, Model, Msg](
   def toVNode(html: Html[Msg]): VNode =
     html match
       case Tag(name, attrs, children) =>
-        val as = js.Dictionary(
+        val as: List[(String, String)] =
           attrs.collect {
             case Attribute(n, v)   => (n, v)
             case NamedAttribute(n) => (n, "")
-          }: _*
-        )
-
-        val props =
-          js.Dictionary(attrs.collect { case Property(n, v) => (n, v) }: _*)
+          }
 
-        val events =
-          js.Dictionary(attrs.collect { case Event(n, msg) =>
-            (n, fun((e: dom.Event) => onMsg(msg.asInstanceOf[dom.Event => Msg](e))))
-          }: _*)
+        val props: List[(String, PropValue)] =
```

Comment:
`PropValue` is an alias to.. `Any`? I realise props can be numbers or strings or booleans, but in HTML land they're all strings in fact? Because they're written into attributes? Might be fine, feels odd.

Reply:
No, I don't think so actually. Props and attributes are different. Of course parseable HTML can only be expressed with attributes, but from JS APIs you'd set the props as their native values.

Reply:
Yep fair enough, I think I was just a bit taken aback to see the Any, but it may be unavoidable.